demosplan/DemosPlanCoreBundle/Entity/Statement/Statement.php

```diff
@@ -4090,16 +4145,45 @@ public function getSimilarStatementSubmitters(): Collection
         return $this->similarStatementSubmitters;
     }
 
+    public function addSimilarStatementSubmitter(ProcedurePerson $similarStatementSubmitter): void
+    {
+        if (!$this->similarStatementSubmitters->contains($similarStatementSubmitter)) {
+            $this->similarStatementSubmitters->add($similarStatementSubmitter);
+        }
+
+        if (!$similarStatementSubmitter->getSimilarForeignStatements()->contains($this)) {
+            $similarStatementSubmitter->addSimilarForeignStatement($this);
+        }
+    }
+
     /**
      * @param Collection<int, ProcedurePerson> $similarStatementSubmitters
      */
     public function setSimilarStatementSubmitters(Collection $similarStatementSubmitters): Statement
     {
+        // clear currently set submitters first because this is setting, not adding
+        foreach ($this->similarStatementSubmitters as $submitter) {
+            $submitter->removeSimilarForeignStatement($this);
+        }
+
+        foreach ($similarStatementSubmitters as $submitter) {
+            $submitter->addSimilarForeignStatement($this); // handle both sites
+//            $this->similarStatementSubmitters->add($submitter);
```

Comment:
I guess the `$submitter` already handles both sides, but uncommenting the line below would make the effect more clear.

Reply:
1d4fd70f984ba66eafc56bd477f334179a3b88c0